.circleci/config.yml

```diff
@@ -19,16 +23,14 @@ jobs:
             curl https://pyenv.run | bash
             pyenv install --skip-existing 3.8.1
             pyenv global 3.8.1
-      - restore_cache:
-          key: v1-deps-cache-{{ checksum "Pipfile.lock" }}
       - run:
           name: Install dependencies
           command: |
             # Doesn't work with --user flag
             pip install pipenv
             pipenv install --dev
       - save_cache:
-          key: v1-deps-cache-{{ checksum "Pipfile.lock" }}
+          key: v1-deps-cache-{{ checksum "Pipfile" }}
```

Comment:
why not to use the lockfile ?

Reply:
It's a lib so there's no lockfile :wink: